### PR TITLE
ROX-25715: add remaining Konflux Slack notifications

### PR DESCRIPTION
**Pull request by tommartensen**
### Description

Apply changes from https://github.com/stackrox/stackrox/pull/12312 to other pipelines.

### User-facing documentation

- [x] CHANGELOG update is not needed
- [x] [documentation PR](https://spaces.redhat.com/display/StackRox/Submitting+a+User+Documentation+Pull+Request)  is not needed

### Testing and quality

- [x] the change is production ready: the change is GA or otherwise the functionality is gated by a feature flag
- [x] CI results are inspected

#### Automated testing

- ~[ ] added unit tests~
- ~[ ] added e2e tests~
- ~[ ] added regression tests~
- ~[ ] added compatibility tests~
- ~[ ] modified existing tests~

Existing tests are sufficient, Konflux only change.

#### How I validated my change

Konflux CI passing.

### CODE REVIEW

File: .tekton/main-build.yaml [hunk 1]
```diff
@@ -60,7 +60,11 @@ spec:
 
   # The pipeline regularly takes >1h to finish.
   timeouts:
-    pipeline: 1h30m0s
+    # The tasks regularly takes 1h to finish.
+    tasks: 1h30m
+    # Reserve time for final tasks to run.
+    finally: 10m
+    pipeline: 1h40m
```
Comment (msugakov) [inline .tekton/main-build.yaml:67]: Please rebase from `master`.

File: .tekton/main-pipeline.yaml [hunk 2]
```diff
@@ -5,6 +5,26 @@ metadata:
...
+    taskRef:
+      params:
+      - name: name
+        value: slack-webhook-notification
+      - name: bundle
+        value: quay.io/konflux-ci/tekton-catalog/task-slack-webhook-notification:0.1@sha256:0dfdfd87a8716ff9c20ae3325eff9a5d52ee9c708959c1e93eaedc852621a4d5
+      - name: kind
+        value: task
+      resolver: bundles
   - name: show-sbom
```
Comment (msugakov) [inline .tekton/main-pipeline.yaml:28]: [nit] Something I realized now: could you please add a blank line between different tasks? As well for basic-component-pipeline.

File: .tekton/main-build.yaml (hunk 1 above)
Comment (tommartensen) [reply .tekton/main-build.yaml]: https://github.com/stackrox/stackrox/pull/12409/commits/e4b4195cbd2f2b7146e736453ad0557a848af319

File: .tekton/main-pipeline.yaml (hunk 2 above)
Comment (tommartensen) [reply .tekton/main-pipeline.yaml]: https://github.com/stackrox/stackrox/pull/12409/commits/6da77b644c9812c044aca908b9904c8dbdcf7731

**Review by msugakov — APPROVED** [review]

**Pull request merged**